docs/samples/Microsoft.ML.Samples/Program.cs

```diff
@@ -10,6 +10,7 @@ public static class Program
 
         internal static void RunAll()
         {
+            
```

Comment:
This can be reverted.

Reply:
Reverted.

Reply:
I still see the changes to this file in the current iteration.

---
In reply to: [328357298](https://github.com/dotnet/machinelearning/pull/4242#discussion_r328357298) [](ancestors = 328357298)

Reply:
Reverted.